docs/genome_to_collection.md

```diff
@@ -0,0 +1,235 @@
+Before proceeding, please ensure that you have an active NERSC account and are logged in to Perlmutter.
+
+```commandline
+ssh perlmutter.nersc.gov
+```
+
+## Step 2: Prepare Source Data Files (FASTA)
+
+### If you already have a KBase narrative with target Genome objects
+If you already possess a KBase narrative containing the desired Genome objects, download the FASTA files associated with
+the Genome objects from the KBase workspace to NERSC. The following command facilitates this process:
+
+```commandline
+# update arguments as needed
+workspace_id=69739
+kbase_collection=ENIGMA
+source_verion=2023.10
+env=CI
+token_filepath=path/to/token_file.txt  # This is a single-line file with the KBase token 
+
+cd /global/cfs/cdirs/kbase/collections/collections
+PYTHONPATH=. python src/loaders/workspace_downloader/workspace_downloader.py \
+    --workspace_id $workspace_id \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_verion \
+    --root_dir $root_dir \
+    --env $env \
+    --token_filepath $token_filepath \
+    --retrieve_sample 
+```
+
+### Download FASTA files from NCBI based on GTDB version
+Alternatively, if you don't possess a KBase narrative, you can download FASTA files from NCBI based on GTDB version 
+using the following command:
+
+```commandline
+# update arguments as needed
+download_file_ext='genomic.fna.gz'
+gtdb_release_ver=214
+
+PYTHONPATH=. python src/loaders/ncbi_downloader/gtdb.py \
+    --download_file_ext $download_file_ext \
+    --release_ver $gtdb_release_ver
+```
+
+### Create KBase Genome objects from Genbank files
+Optionally, you can also download Genbank files from NCBI and create KBase Genome objects from them using the 
+following command. This process will also retrieve the FASTA file from the created Genome objects.
+
+```commandline
+# update arguments as needed
+
+# download Genbank files from NCBI based on GTDB version
+download_file_ext='genomic.gbff.gz'
+gtdb_release_ver=214
+
+PYTHONPATH=. python src/loaders/ncbi_downloader/gtdb.py \
+    --download_file_ext $download_file_ext \
+    --release_ver $gtdb_release_ver
+    
+# create KBase Genome objects from the downloaded Genbank files
+workspace_id=72231
+kbase_collection=GTDB
+source_verion=$gtdb_release_ver
+load_id=1
+au_service_ver=dev
+gfu_service_ver=dev
+cbs_max_tasks=10
+
+PYTHONPATH=. python src/loaders/workspace_uploader/workspace_uploader.py \
+    --workspace_id $workspace_id \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_verion \
+    --env $env \
+    --token_filepath $token_filepath \
+    --au_service_ver $au_service_ver \
+    --gfu_service_ver $gfu_service_ver \
+    --cbs_max_tasks $cbs_max_tasks \
+    --load_id $load_id  
+```
+
+# Step 3: Execute tools on FASTA files
+
+Once the source data files are prepared, execute tools on these FASTA files. You can choose from various available 
+tools such as gtdb_tk, checkm2, microtrait, or mash.
```

Comment:
Should note here that this will consume NERSC allocation

Reply:
👍